### ISSUE TITLE
Benchmark shows that Android sqlite's read performance better than requery's

### ISSUE THREAD

**Issue by Tougee** [requery/sqlite-android#128]
OnePlus A5010
Android 9

```
2020-03-12 21:14:23.834 2151-2186/io.requery.android.sqlite.test I/SQLite: Android: Read AVG 42.4 Write AVG 306.4
     Rows/sec 235849.05 Inserts/sec 32637.078
2020-03-12 21:14:23.835 2151-2186/io.requery.android.sqlite.test I/SQLite: requery: Read AVG 45.6 Write AVG 212.0
     Rows/sec 219298.25 Inserts/sec 47169.812
2020-03-12 21:16:20.465 2257-2301/io.requery.android.sqlite.test I/SQLite: Android: Read AVG 42.6 Write AVG 299.6
     Rows/sec 234741.8 Inserts/sec 33377.84
2020-03-12 21:16:20.466 2257-2301/io.requery.android.sqlite.test I/SQLite: requery: Read AVG 45.4 Write AVG 218.4
     Rows/sec 220264.31 Inserts/sec 45787.547
2020-03-12 21:20:20.244 2392-2418/io.requery.android.sqlite.test I/SQLite: Android: Read AVG 42.6 Write AVG 311.8
     Rows/sec 234741.8 Inserts/sec 32071.842
2020-03-12 21:20:20.244 2392-2418/io.requery.android.sqlite.test I/SQLite: requery: Read AVG 49.2 Write AVG 214.8
     Rows/sec 203252.03 Inserts/sec 46554.934
2020-03-12 21:21:52.682 2575-2602/io.requery.android.sqlite.test I/SQLite: Android: Read AVG 43.0 Write AVG 280.2
     Rows/sec 232558.14 Inserts/sec 35688.793
2020-03-12 21:21:52.682 2575-2602/io.requery.android.sqlite.test I/SQLite: requery: Read AVG 44.0 Write AVG 214.2
     Rows/sec 227272.72 Inserts/sec 46685.34
```

**Comment by mikehardy** [comment]
Sure, it's gotten better and better. I support API15 devices though. Benchmark every API currently supported by the play store, API15 through API30 then I'm interested. Especially pay attention to a feature matrix of what requery supports vs (for example API15 android built-in sqlite). Is the feature you need supported? Are you sure? With requery you are sure :-) - that's the point, not raw speed I think

**Comment by Tougee** [comment]
Ok, take it easy, the new features supported by requery are very useful, I'm just looking for a FTS5 solution for my Android project and confusing about the benchmark result on my specific device.

**Comment by mikehardy** [comment]
lies, damned lies, and benchmarks ;-). One nice thing about requery is you can build it yourself. Perhaps if you bisect different releases here you get different results, and perhaps the results may be explained by different makefile configurations (more or less options)

**Comment by crossle** [comment]
Should update the library readme benchmark, it's just compare Android 4.4, compare with Android 10, or 11?

**Comment by mikehardy** [comment]
@crossle my guess is there will be nearly no performance difference at this point, on modern android
the main benefit at this point is precise control of which version (if that is important to you) and which extras are included in the build or not (like JSON extensions or what have you)

**Comment by npurushe** [comment]
@mikehardy 's explanation is pretty spot on. Originally the primary benefit of the library was for older versions (mostly pre 8.0) of android. That then expanded to compiling in some additional SQLite features that Android doesn't support. Any performance differences on newer versions should come down to compiler flags being used and be pretty minimal but that is worth confirming. 
The readme will be updated to reflect this thanks!

**Comment by benjdero** [comment]
I've ran a bunch of benchmarks on 5 different devices with various versions of Android, ranging from 5.1 to 11.

Surprisingly, I noticed a far better performance difference on the newest devices. My device running Android 5.1 only show a 10% increased performance using Requery's embedded SQLite compared to Android "native" SQLite, meanwhile the Pixel 4A running on Android 11 show a 90% increased performance.
